modules/ossm-control-plane-configuration-migration-requirements.adoc

```diff
@@ -36,14 +36,14 @@ For both control planes, during the migration ensure that network policies do no
 * The data plane namespaces and the control plane
 * The data plane namespaces themselves
 
-In the premigration checklist, you are instructed to disable network policies. However, you can manually re-create them. Manually created network policies must allow traffic for both control planes. When a data plane namespace is migrated to 3.0, the `maistra.io/member-of` label is automatically removed. Do not use this label in network policies. For more information, see "Migration of Network Policies".
+In the premigration checklist, you are instructed to disable network policies. However, you can manually re-create them. Manually created network policies must allow traffic for both control planes. When a data plane namespace is migrated to 3.0, the `maistra.io/member-of` label is automatically removed. Do not use this label in network policies. For more information, see "Set up network policies to use during migration".
 
 [NOTE]
 ====
 In {SMProduct} 2.6 installations, the `maistra.io/member-of` label is automatically created. This label cannot be used because it is automatically removed during the migration process.
 ====
 
-Incorrectly configured network policies can disrupt mesh traffic. When you run the migration, be careful when you create network policies to prevent traffic disruption. For more information, see "Network Policy migration".
+Incorrectly configured network policies can disrupt mesh traffic. When you run the migration, be careful when you create network policies to prevent traffic disruption. For more information, see Set up network policies to use during migration".
```

Comment:
```suggestion
Incorrectly configured network policies can disrupt mesh traffic. When you run the migration, be careful when you create network policies to prevent traffic disruption. For more information, see "Set up network policies to use during migration".
```

Reply:
Thanks @xenolinux ! Made the edit and squashed the PR.